cmd/internal/su.go

```diff
@@ -47,3 +41,48 @@ func AutoSu() {
 	}
 	os.Exit(stat.ExitCode())
 }
+
+func trySudo() (path string, arg []string) {
+	pathSudo, err := exec.LookPath("sudo")
+	if err != nil || !isExistAndExecutable(pathSudo) {
+		return "", nil
+	}
+	// https://github.com/WireGuard/wireguard-tools/blob/71799a8f6d1450b63071a21cad6ed434b348d3d5/src/wg-quick/linux.bash#L85
+	return pathSudo, []string{
+		pathSudo,
+		"-E",
+		"-p",
+		fmt.Sprintf("%v must be run as root. Please enter the password for %%u to continue: ", filepath.Base(os.Args[0])),
+		"--",
+	}
+}
+
+func tryPolkit() (path string, arg []string) {
+	var possible = []string{"pkexec"}
+	for _, v := range possible {
+		path, err := exec.LookPath(v)
+		if err != nil {
+			continue
+		}
+		if isExistAndExecutable(path) {
+			switch v {
+			case "pkexec":
```

Comment:
`run0` doesn't need `--keep-cwd` argument as it defaults to the client's current working directory.

```suggestion
            case "run0":
                    return path, []string{path}
			case "pkexec":
```